### PR TITLE
add kissui.scrollanim v0.2.0 w/ npm auto-update

### PR DESCRIPTION
**Pull request by maruilian11**
**npm package url:** https://www.npmjs.com/package/kissui.scrollanim
**git repo url:** https://github.com/usablica/kissui.scrollanim
**Watch 16
  Star 130
 Fork 50**
@Amomo could u help me check this PR for #8735 ? thank you!
**download from https://github.com/usablica/kissui.scrollanim/archive/v0.1.1.zip**

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR 
 * [x] The lib has notable popularity 
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry 
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update) 
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by Amomo** [comment]
@maruilian11 I think it should be #8735 and https://github.com/usablica/kissui.scrollanim in your PR comment. Could you revise it? The remaining parts are fine.

**Comment by maruilian11** [comment]
@Amomo thank you! i corrected it and rebased the commit with our latest master branch.

File: ajax/libs/kissui.scrollanim/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "dependencies": {
+    "kissui.position": "^0.1.0"
+  },
+  "author": "Afshin Mehrabani",
+  "license": "MIT",
+  "homepage": "http://scrollanim.kissui.io/",
+  "npmName": "kissui.scrollanim",
+  "npmFileMap": [
+    {
+      "basePath": "",
```
Comment (PeterDaveHello) [inline ajax/libs/kissui.scrollanim/package.json]: Why don't you use the `build` folder?

**Comment by maruilian11** [comment]
@Amomo i modify `basepath` to build, could you help me check this PR again? thank you!

**Comment by PeterDaveHello** [comment]
@maruilian11 So what is the reason you didnt use it originally?

**Comment by maruilian11** [comment]
@peterdavehello because I saw the git repository first, there are not css and js we want in build folder. then, I saw npm package structure, I thought they are the same files under build folder, so I didn't check it.

i will check it next time.

File: ajax/libs/kissui.scrollanim/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  "author": "Afshin Mehrabani",
+  "license": "MIT",
+  "homepage": "http://scrollanim.kissui.io/",
+  "npmName": "kissui.scrollanim",
+  "npmFileMap": [
+    {
+      "basePath": "build",
+      "files": [
+        "scrollanim*"
```
Comment (PeterDaveHello) [inline ajax/libs/kissui.scrollanim/package.json]: So why don't we use `"**/*"` ?

**Comment by maruilian11** [comment]
@Amomo i have changed filemap to `**/*` and update the version to `0.2.0`, could you help me check this PR again? thank you!

**Pull request merged**